MapboxNavigation/NavigationViewController.swift

```diff
@@ -373,6 +399,26 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         if routeProgress.currentLegProgress.alertUserLevel == .arrive {
             navigationDelegate?.navigationViewController?(self, didArriveAt: destination)
         }
+        
+        applyStyle()
```

Comment:
Can we optimize this a bit? Perhaps we could get `intervalUntilTimeOfDayChange` when we present the NavigationViewController and only fire it once when it should happen?